pdf-reader.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency("morecane")
   spec.add_development_dependency("ir_b")
   spec.add_development_dependency("rdoc")
+  spec.add_development_dependency("test-unit")
```

Comment:
is this necessary? I don't think `test-unit` is required on master as of a few days ago